test/test.lua

```diff
@@ -1507,10 +1515,10 @@ math.randomseed(os.time())
 mytester = torch.Tester()
 mytester:add(cudnntest)
 
-if torch.random(1,2) == 1 then
-   cudnn.benchmark = true -- run manual auto-tuner
---   cudnn.verbose = true
-end
+-- if torch.random(1,2) == 1 then
```

Comment:
Sure! However, one of the reasons I have uncommented the condition is that it's always false: random generator needs to be seeded probably?